db/migrations/20250130172009_add_canary_steps_to_deployments.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:deployments) do
+      # TODO: what size?
+      add_column :canary_steps, String, size: 4096
```

Comment:
thought: Should this be a JSON column for Postgres?

Reply:
MySQL 5.7.8 and above  [supports json columns](https://medium.com/@nagarjoon.b/the-json-data-type-in-mysql-pluses-and-minuses-d4f98b9882d7#:~:text=Since%20version%205.7.,this%20more%20traditional%20data%20format.). Though not entirely sure we can assume someone is using that version while we still support MySQL 5.7.x  https://github.com/cloudfoundry/cloud_controller_ng/issues/3949. I don't think we document a patch release of mysql

We could do this just for postgres but I'm not sure theres much benefit for added delta